apps/03TicTacToe.js

```diff
@@ -23,32 +26,61 @@ function printBoard() {
 
 function horizontalWin() {
     // Your code here
+    if ((board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) ||
+        (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn) ||
+        (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn))
+
+        return true;
 }
 
 function verticalWin() {
     // Your code here
+    if ((board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn) ||
+        (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn) ||
+        (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn))
+
+        return true;
 }
 
 function diagonalWin() {
     // Your code here
+    if ((board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn) ||
+        (board[0][2] === playerTurn && board[1][1] === playerTurn && board[2][0] === playerTurn))
+
+        return true;
 }
 
 function checkForWin() {
     // Your code here
+    if ((horizontalWin() === true) || (verticalWin() === true) || (diagonalWin() === true))
+
+      return true;
 }
 
 function ticTacToe(row, column) {
     // Your code here
+    board[row][column] = playerTurn;
+    if (checkForWin() === true){
```

Comment:
instead of (checkForWin()===true), you can simply check if(checkForWin()). Abstraction :)